clients/docs/java-springboot.rst

```diff
@@ -0,0 +1,176 @@
+.. _client-examples-java-springboot:
+
+Java Spring Boot
+================
+
+In this tutorial, you will run a Java Spring Boot client application that
+produces messages to and consumes messages from an |ak-tm| cluster.
+
+.. include:: includes/client-example-overview.rst
+
+Prerequisites
+--------------
+
+Client
+~~~~~~
+
+-  Java 1.8.
+
+
+Kafka Cluster
+~~~~~~~~~~~~~
+
+.. include:: includes/client-example-prerequisites.rst
+
+
+Setup
+-----
+
+#. .. include:: includes/clients-checkout.rst
+
+#. Change directory to the example for Java Spring Boot.
+
+   .. code-block:: bash
+
+      cd clients/cloud/java-springboot/
+
+#. .. include:: includes/client-example-create-file-java.rst
```

Comment:
We need a new include for springboot ;)  Let's call it `includes/client-example-create-file-springboot.rst`?